tools/mason/Makefile

```diff
@@ -0,0 +1,45 @@
+
+
+ifndef CHPL_MAKE_HOME
+export CHPL_MAKE_HOME=$(realpath $(shell pwd)/../..)
+endif
+
+include $(CHPL_MAKE_HOME)/make/Makefile.base
+
+bdir=$(CHPL_BIN_DIR)
+link=$(bdir)/mason
+
+
+REGISTRY=https://github.com/Spartee/mason-registry
+
+all: mason
+
+mason:
+	@echo "Building Mason..."
+	#build
+	#install
+
+build:
+	@mkdir -p ~/.mason
+	@mkdir -p ~/.mason/src
+	@git -C ~/.mason/ clone $(REGISTRY) registry
+	@echo "Building Mason..."
+	@chpl mason.chpl
```

Comment:
Does this assume `TOML.chpl` is in `modules/packages`? Perhaps this should (temporarily) pass a `-M test/modules/packages/TOML` until then.

@benharsh aptly pointed out that we'll need to promote `TOML.chpl` to `modules/packages/` by the 1.16 release (9/18), because `test/` is not included in the release tarball.